graveler/committed/committed.proto

```diff
@@ -8,4 +8,5 @@ message RangeData {
 	bytes min_key = 1;
 	bytes max_key = 2;
 	uint64 estimated_size = 3;
+	uint32 count = 4;
```

Comment:
```suggestion
	uint32 num_records = 4;
```
or
```suggestion
	uint32 records_count = 4;
```
or similar: "count" is units but not what is being counted.

Reply:
Also consider
```suggestion
	int64 count = 4;
```
a. Lets you stay signed (so safe arithmetic is safer)
b. We can expect customers to be within 1000x of this once we have a big user; why be close to the limit?
c. Varint encoding makes it free in disk space.

Reply:
Changing the uint32 to int64.
I think "count" is descriptive enough and in accordance with the structs in our code.